## Workshop Resources
### Session 1: Machine Learning for Systems Serology Data
**Goal:** We will introduce a general machine learning workflow to deal with system serology datasets.
#### Before workshop, please review the items below:
1. [Readme.md](SystemSerology_MachineLearning/README.md): Instructions for installation of required R packages.
2. [Notebook 1](https://github.com/cvisb/2020_systems-biology_workshop/blob/master/SystemSerology_MachineLearning/systemsSerology_part1.Rmd): Dataset
### Session 2a: B Cell Repertoire Sequencing
[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/briney/cvisb_workshop_2020/master?urlpath=lab/tree/tutorial)  

**Goal:** We will walk through several examples of BCR sequence analysis, using two different sequence datasets. Our goal is to introduce programming primitives and tools to help annotate, discover and visualize BCR sequence data.
#### Before workshop, please review the items below:
1. All you need is a computer with a modern browser (Chrome/Brave preferred). No additional programs or dependencies need to be installed. MacOS, Windows or Linux computers are all acceptable.
2. Once you arrive at the tutorial, click the "launch binder" badge above to launch your Binder instance. This takes about 30 seconds or a minute. If you sit and stare at the spinning Binder icon the whole time, it will feel like an hour. Once started, you'll be able to independently run and modify the code used in the tutorial.
3. At the top of each open Jupyter Lab notebook, there's a "Download" button. Clicking it will download the current notebook to your computer. If you'd like to peruse the [GitHub repo](https://github.com/briney/cvisb_workshop_2020) for this tutorial, you can click the "GitHub" button in any open notebook or the link right ........... up ........... there :point_up:
4. The second half of the B Cell Repertoire Sequencing tutorial will demonstrate how to make several of the figure panels from [this paper](https://immunology.sciencemag.org/content/2/14/eaan6809.long).
### Session 2b: Identification and Charaterization of T cell Subsets from Single Cell Data
**Goal:** We will introduce a general workflow to deal with single cell RNA-seq data from peripheral blood mononuclear cells, with a goal on identifying and characterizing T cell clusters.
#### Before workshop, please review the items below:
1. [Readme.md](https://github.com/watronfire/CViSB_Workshop_TCells/blob/b70de631c16d0696938cbb09e8147a4d1be9d4a9/README.md): Instructions for installation of required Python and R packages.
2. Dataset: Anndata h5ad file, Zanini et al. PNAS 2018. (Dengue infection vs. Healthy). Available as download from [Anderson Lab's Google Cloud](https://storage.cloud.google.com/andersen-lab_temp/dengue_all.hdf5), or on flashdrive during workshop. Move this file into the data folder of GitHub repository.
