## Workshop Resources
### Session 1: Machine Learning for Systems Serology Data
**Goal:** We will introduce a general machine learning workflow to deal with system serology datasets.
#### Before workshop, please review the items below:
1. [Readme.md](SystemSerology_MachineLearning/README.md): Instructions for installation of required R packages.
2. [Notebook 1](https://github.com/cvisb/2020_systems-biology_workshop/blob/master/SystemSerology_MachineLearning/systemsSerology_part1.Rmd): Dataset
### Session 2a: B Cell Repertoire Sequencing
**Goal:** We will walk through several examples of BCR sequence analysis, using datasets from hundreds to millions of sequences. Our goal is to introduce programming primitives and tools to help annotate, discover and visualize BCR sequence data.
#### Before workshop, please review the items below:
1. All you need is a computer with a modern browser (Chrome/Brave preferred). No additional programs or dependencies need to be installed. MacOS, Windows or Linux computes are all acceptable.
2. A placeholder landing site for the tutorial can be found [HERE](https://mybinder.org/v2/gh/briney/cvisb_workshop_2020/placeholder)
### Session 2b: Identification and Charaterization of T cell Subsets from Single Cell Data
**Goal:** We will introduce a general workflow to deal with single cell RNA-seq data from peripheral blood mononuclear cells, with a goal on identifying and characterizing T cell clusters.
#### Before workshop, please review the items below:
1. [Readme.md](https://github.com/watronfire/CViSB_Workshop_TCells/blob/3a961cc3de6daaff4bded8b9130236da99bdcc17/README.md): Instructions for installation of required Python and R packages.
2. Dataset: Anndata h5ad file, Zanini et al. PNAS 2018. (Dengue infection vs. Healthy)
3. Load notebook: Binder link coming soon!
